src/agent.js

```diff
@@ -2,179 +2,182 @@
  * This package handles bot messages from dialog flow
  */
 
-const { WebhookClient, Text, Card, Image, Suggestion, Payload } = require('dialogflow-fulfillment');
-const i18n = require('i18n');
-const { getTelegramButtons } = require('./misc');
-const { listApps, addApp, rmApp, subscribeToApp, unsubscribeToApp } = require('./db');
+import path from 'path'
+/* eslint-disable no-unused-vars */
+const { WebhookClient, Text, Card, Image, Suggestion, Payload } = require('dialogflow-fulfillment')
+/* eslint-enable no-unused-vars */
+const i18n = require('i18n')
+const { getTelegramButtons } = require('./misc')
+const { listApps, addApp, rmApp, subscribeToApp, unsubscribeToApp } = require('./db')
 const { LANGUAGE } = require('./environment')
-const intentMap = new Map();
+const intentMap = new Map()
 
 i18n.configure({
-    directory: __dirname + '/locales',
-    defaultLocale: LANGUAGE,
-    register: global
+	directory: path.join(__dirname, '/locales'),
+	defaultLocale: LANGUAGE,
+	register: global
 })
 
 const actions = [
-    __("List all apps being monitored"),
-    __("Subscribe to alerts"),
-    __("Unsubscribe to alerts"),
-    __("Add a new app"),
-    __("Remove an app"),
-    __("Change an app's bb-promster address"),
-    __("Help with setting up my app observation cluster")
-];
+	__('List all apps being monitored'),
+	__('Subscribe to alerts'),
+	__('Unsubscribe to alerts'),
+	__('Add a new app'),
+	__('Remove an app'),
+	__("Change an app's bb-promster address"),
```

Comment:
@gutorc92, a little bit of a OCD here, but why all other string are with simple quotes and this one is with double quotes?